internal/controller/config/config.go

```diff
@@ -53,6 +53,10 @@ func (*ConfigManager) GetControllerDefinitionsMgr() *ControllerDefinitionsManage
 type GlobalConfig struct {
 	LogLevel          int    `json:"logLevel" mapstructure:"logLevel"`
 	IncludedResources string `json:"includedResources" mapstructure:"includedResources"`
+	// Feature flag - if enabled causes pipeline(s) to be paused when pipeline, numaflow controller, or ISB Service gets updated
+	DataLossPrevention bool `json:"dataLossPrevention" mapstructure:"dataLossPrevention"`
+	// List of Numaflow Controller image names to look for
+	NumaflowControllerImageNames []string `json:"numaflowControllerImageNames" mapstructure:"numaflowControllerImageNames"`
```

Comment:
We need to add unit test for loading these two new fields right?

Reply:
do you mean config_test.go? Do you see any real chance of this not loading? In general, I think unit tests should be for complicated areas of code, right?

Reply:
(I did start working in a separate branch on adding unit tests for the reconcile() functions, though)

Reply:
this is just to make sure the parsing/loading is correct, e.g adding these fields for verification in https://github.com/numaproj/numaplane/blob/340cb90285797be8e49a056298ed8e5c664eaad3/internal/controller/config/config_test.go#L27.